### PR TITLE
DOCSUP-7045: Edit and translate to Russian (Upgrading CH and https interserver replication)

### PR DESCRIPTION
**Pull request by ka1bi4**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #19338 and #20015

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Added documentation:
- Upgrading ClickHouse.
- HTTPS interserver replication settings.

### CODE REVIEW

File: docs/ru/operations/update.md [hunk 1]
```diff
@@ -15,4 +15,19 @@ $ sudo service clickhouse-server restart
 
 Если ClickHouse установлен не из рекомендуемых deb-пакетов, используйте соответствующий метод обновления.
 
-ClickHouse не поддерживает распределенное обновление. Операция должна выполняться последовательно на каждом отдельном сервере. Не обновляйте все серверы в кластере одновременно, иначе кластер становится недоступным в течение некоторого времени.
+!!! warning "Предупреждение"
+    ClickHouse не поддерживает распределенное обновление. Операция обновления должна выполняться последовательно на каждом отдельном сервере. Не обновляйте все серверы в кластере одновременно, иначе кластер станет недоступен в течение некоторого времени.
+
+Обновление ClickHouse до определенной версии:
+
+Пример:
```
Comment (adevyatova) [inline docs/ru/operations/update.md:23]:
```suggestion
**Пример**
```

File: docs/en/operations/update.md [hunk 2]
```diff
@@ -15,7 +15,8 @@ $ sudo service clickhouse-server restart
 
 If you installed ClickHouse using something other than the recommended `deb` packages, use the appropriate update method.
 
-ClickHouse does not support a distributed update. The operation should be performed consecutively on each separate server. Do not update all the servers on a cluster simultaneously, or the cluster will be unavailable for some time.
+!!! warning "Warning"
+    ClickHouse does not support a distributed update. The operation should be performed consecutively on each separate server. Do not update all the servers on a cluster simultaneously, or the cluster will be unavailable for some time.
```
Comment (alexey-milovidov) [inline docs/en/operations/update.md:19]: This is wrong. You can update multiple servers at once as soon as there is no moment of time when all replicas of one shards are offline.

File: docs/ru/operations/server-configuration-parameters/settings.md [hunk 3]
```diff
@@ -297,11 +297,36 @@ ClickHouse проверяет условия для `min_part_size` и `min_part
 <interserver_http_host>example.yandex.ru</interserver_http_host>
 ```
 
+## interserver_https_port {#interserver-https-port}
+
+Порт для обмена данными между серверами ClickHouse по протоколу `HTTPS`.
```
Comment (alexey-milovidov) [inline docs/ru/operations/server-configuration-parameters/settings.md:302]: Между репликами.

File: docs/ru/operations/server-configuration-parameters/settings.md [hunk 4]
```diff
@@ -297,11 +297,36 @@ ClickHouse проверяет условия для `min_part_size` и `min_part
...
+
+**Пример**
+
+``` xml
+<interserver_https_port>9010</interserver_https_port>
+```
+
+## interserver_https_host {#interserver-https-host}
+
+Имя хоста, которое могут использовать другие серверы для обращения к этому хосту по протоколу `HTTPS`.
```
Comment (alexey-milovidov) [inline docs/ru/operations/server-configuration-parameters/settings.md:312]: Реплики.

File: docs/ru/operations/update.md [hunk 5]
```diff
@@ -15,4 +15,19 @@ $ sudo service clickhouse-server restart
 
 Если ClickHouse установлен не из рекомендуемых deb-пакетов, используйте соответствующий метод обновления.
 
-ClickHouse не поддерживает распределенное обновление. Операция должна выполняться последовательно на каждом отдельном сервере. Не обновляйте все серверы в кластере одновременно, иначе кластер становится недоступным в течение некоторого времени.
+!!! warning "Предупреждение"
+    ClickHouse не поддерживает распределенное обновление. Операция обновления должна выполняться последовательно на каждом отдельном сервере. Не обновляйте все серверы в кластере одновременно, иначе кластер станет недоступен в течение некоторого времени.
```
Comment (alexey-milovidov) [inline docs/ru/operations/update.md:19]: Неверно.

File: docs/ru/operations/server-configuration-parameters/settings.md [hunk 6]
```diff
@@ -297,11 +297,36 @@ ClickHouse проверяет условия для `min_part_size` и `min_part
...
+
+**Пример**
+
+``` xml
+<interserver_https_port>9010</interserver_https_port>
+```
+
+## interserver_https_host {#interserver-https-host}
+
+Имя реплики, которое могут использовать другие реплики для обращения к ней по протоколу `HTTPS`.
```
[the remark this answers is not in the document]
Comment (ka1bi4) [reply docs/ru/operations/server-configuration-parameters/settings.md:312]: > 

Тут как раз изначально был хост, и поступило замечание исправить на реплику https://github.com/ClickHouse/ClickHouse/pull/21559#discussion_r602982203

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**